docs/reference/how-to/search-speed.asciidoc

```diff
@@ -9,6 +9,27 @@ fast. In general, you should make sure that at least half the available memory
 goes to the filesystem cache so that Elasticsearch can keep hot regions of the
 index in physical memory.
 
+[discrete]
+=== Avoid page cache thrashing by using modest read ahead values on Linux
+
+Search can cause a lot of randomized read IO. When using the default
+Elasticsearch filestore (fs/<<hybridfs,hybridfs>>), Lucene files using memory
+mapping may contribute to unnecessary read I/O when the underlying block device
+has a high readahead value.
+
+Most Linux distributions use a sensible readahead value of `128KiB` for a
+single plain device, however, when using software raid, LVM or dm-crypt the
```

Comment:
I don't think that `dm-crypt` by itself does that, but given that any combination of those maybe used, I am being defensive here and raising awareness that it is important to check the current readahead setting of the block device backing Elasticsearch's `path.data`